<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-GQPKBG72" failures="0" tests="1" name="com.ibm.Eapp.auto.Controller" time="7.203" errors="1" timestamp="27 Oct 2022 19:04:23 GMT">
  <testcase classname="com.ibm.Eapp.auto.Controller" name="reDirectProcess" time="7.203">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.relevantcodes.extentreports.ExtentTest.isPathRelative(ExtentTest.java:502)
	at com.relevantcodes.extentreports.ExtentTest.addScreenCapture(ExtentTest.java:287)
	at com.ibm.utility.reports.TestReporter.Log_Fail(TestReporter.java:161)
	at com.ibm.automation.ui.AutomationWrapper.Execute(AutomationWrapper.java:377)
	at com.ibm.automation.ui.AutomationWrapper.Execute(AutomationWrapper.java:273)
	at com.ibm.automation.ui.AutomationWrapper.Execute(AutomationWrapper.java:138)
	at com.ibm.Eapp.mod.EappCSEMod.Execute(EappCSEMod.java:121)
	at com.ibm.Eapp.auto.Controller.reDirectProcess(Controller.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at com.ibm.automation.ui.Create_TestNGXML.runTestNG(Create_TestNGXML.java:281)
	at com.ibm.automation.ui.Create_TestNGXML.createXMLfile(Create_TestNGXML.java:99)
	at com.ibm.Eapp.xml.TestNGXML_Helper.createXMLfile(TestNGXML_Helper.java:46)
	at com.ibm.Eapp.run.ipaRunner.main(ipaRunner.java:22)
]]>
    </error>
  </testcase> <!-- reDirectProcess -->
</testsuite> <!-- com.ibm.Eapp.auto.Controller -->
